velox/common/memory/MemoryUsageTracker.cpp

```diff
@@ -223,29 +223,31 @@ void MemoryUsageTracker::maybeUpdatePeakBytes(int64_t newPeak) {
 }
 
 void MemoryUsageTracker::checkNonNegativeSizes(const char* errmsg) const {
-  // TODO: make these CHECK failures after making usage tracker thread-safe.
-  if (reservationBytes_ < 0) {
-    LOG_EVERY_N(ERROR, kLogEveryN) << "MEMORY: Negagtive reservation bytes "
-                                   << errmsg << " " << reservationBytes_;
-  }
+  VELOX_CHECK_GE(
+      reservationBytes_,
+      0,
+      "Negative reservationBytes_ {}, {}",
```

Comment:
VELOX_CHECK_GE will include the value of reservationBytes_ in the error message automatically, hence, no need to include it again in the error message.

`"Negative reservationBytes_: {}", toString()`